custom_connectors/basic_auth/transmit_sms_connector.rb

```diff
@@ -115,6 +115,17 @@
             control_type: "text-area",
             optional: false
           },
+          {
+            name: "tracked_link_url",
+            label: "Tracked Link",
```

Comment:
Generally all labels, we follow the convention as Start with Upper case followed by all lower case.